src/charm.py

```diff
@@ -210,6 +211,14 @@ def _bootstrap_mysqlrouter(self) -> bool:
 
             self.unit_peer_data[UNIT_BOOTSTRAPPED] = "true"
 
+            try:
+                subprocess.check_call(["open-port", "6446/tcp"])
+                subprocess.check_call(["open-port", "6447/tcp"])
+                subprocess.check_call(["open-port", "6448/tcp"])
+                subprocess.check_call(["open-port", "6449/tcp"])
+            except subprocess.CalledProcessError:
+                logger.exception("failed to open port")
```

Comment:
In addition, for kubernetes, I wonder if we should block the charm when some port fails to be exposed.